client/js/views/templates/Footer.js

```diff
@@ -1,14 +1,88 @@
-module.exports = function() { return `` +
-`<footer>
-    <div>
-        <span>Contact</span>
-        <span>
-            <a href="mailto:info@futuredays.io">info@futuredays.io</a>
-        </span>
-    </div>
-    <div>
-        <span>Copyright</span>
-        <span>${new Date().getFullYear()} FutureDays Software</span>
-    </div>
-</footer>`
+/**
+* Project: Future Days Farm Website
+* Created By: Chris Baron
+* Date Last Modified: 1/19/2018 by Alex Cadigan
+* Description: This file builds the footer elements on the Farm Website homepage
+*/
+
+module.exports = function() 
+{ 
+    return `<footer id = "footerBody">
+
+    <!-- Future Days Farm Info -->
+    <div id = "footerTitle" class = "footer">
+
+        FUTURE DAYS FARM
+
+    </div>
+
+    <div class = "footer">
+
+        2123 Tiny Road
+        <br>
+        Town Name, Michigan 33344        
+
+    </div>
+
+    <br>
+
+    <div class = "footer">
+
+        <a id = "footerLink" href = "mailto:Info@FutureDaysFarm.com"> Info@FutureDaysFarm.com </a>
+        <br>
+        (333) 323-8899
+
+    </div>
+
+    <br>
+
+    <!-- Copyright -->
+    <div id = "footerBottom" class = "footer">
+
+        Copyright ${ new Date().getFullYear() } FutureDays Software
+
+    </div>
+
+    <style>
```

Comment:
Put styles in the `stylus/include` directory.  See the Hazy repo for examples of stylus syntax.  Put the fonts in `stylus/include/base`.

Reply:
The template files should mostly be HTML, with `data-js` used for manipulation.

Also, try to use less white spaces in these files.  I still need to add a script that removes all whitespace in order to make things smaller.